cmd/controller/main.go

```diff
@@ -330,6 +338,7 @@ func parseEnvFlags() config {
 	runtime.Must(viper.BindEnv(logSizeLimitMBFlag))
 	runtime.Must(viper.BindEnv(allocationBatchWaitTime))
 	runtime.Must(viper.BindPFlags(pflag.CommandLine))
+	runtime.Must(viper.BindEnv(leaderElectionFlag))
```

Comment:
reorder with 340 to keep the "meta" binds at the bottom.

Reply:
Done